### PR TITLE
VPN-2035: Add 7-day trial label to mobile onboarding

### PR DESCRIPTION
**Pull request by MattLichtenstein**
## Description

- Added a 7-day trial label to the mobile onboarding view

## Reference

[VPN-2035: As a prospective VPN subscriber,  I want to be able to see that a 7 day free trial is available while I am learning about Mozilla VPN in the onboarding panel](https://mozilla-hub.atlassian.net/browse/VPN-2035)

## Checklist
    
- [x] My code follows the style guidelines for this project
- [x] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [x] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: src/ui/views/ViewMobileOnboarding.qml [hunk 1]
```diff
@@ -256,18 +256,27 @@ VPNFlickable {
...
             Layout.preferredWidth: parent.width
             spacing: VPNTheme.theme.windowMargin
 
+            VPNInterLabel {
+                Layout.maximumWidth: parent.width
+                Layout.alignment: Qt.AlignCenter
+
+                text: VPNl18n.FreeTrialsStartYourFreeTrial
+                color: VPNTheme.colors.white
+            }
```
Comment (flozia) [inline src/ui/views/ViewMobileOnboarding.qml:270]: From the [Figma specs](https://www.figma.com/file/kpz1dqP2terBMIZLjiOhev/Free-Trials?node-id=109%3A289) it seems like the font style should supposed to be _Inter SemiBold 600_ and the color not plain white.

**Review by flozia — CHANGES_REQUESTED** [review]
The placement of the label looks good to me. Regarding the font style: We currently do not use Inter SemiBold 600 elsewhere, but it seems like the time has come to include it.

Can we just add new font styles to the project or is there anything we have to consider @lesleyjanenorton?

cc @peterbenvenuto

File: src/ui/views/ViewMobileOnboarding.qml (hunk 1 above)
Comment (MattLichtenstein) [reply src/ui/views/ViewMobileOnboarding.qml:270]: Correct, we don't use Inter Semibold 600 or white.80 (#FFFFFFCC) anywhere else - will double check with @peterbenvenuto

**Review by flozia — APPROVED** [review]
Thank you for adding the new font style!

File: src/ui/views/ViewMobileOnboarding.qml [hunk 2]
```diff
@@ -256,18 +256,28 @@ VPNFlickable {
...
             Layout.preferredWidth: parent.width
             spacing: VPNTheme.theme.windowMargin
 
+            VPNInterLabel {
+                Layout.maximumWidth: parent.width
+                Layout.alignment: Qt.AlignCenter
+
+                text: VPNl18n.FreeTrialsStartYourFreeTrial
+                color: VPNTheme.colors.white80
+                font.family: theme.fontInterSemiBoldFamily
```
[the remark this answers is not in the document]
Comment (MattLichtenstein) [reply src/ui/views/ViewMobileOnboarding.qml:270]: oh wow 🤦 nice catch

**Pull request merged**